### PR TITLE
June 2022 Updates

### PR DESCRIPTION
**Pull request by steveoh**
Updating the tool to reflect recent data changes in FacilityUST, Soils, and Census

### CODE REVIEW

File: src/tank_risk_tool.pyt [hunk 1]
```diff
@@ -135,9 +135,9 @@ class TankResult():
         'lakes_nhd': LayerAttributes(DISTANCE, 'lakesVal', 'lakeSev', 'lakesVal', 'lakeSev'),
         'streams_nhd': LayerAttributes(DISTANCE, 'streamsVal', 'streamsSev', 'streamsVal', 'streamsSev'),
         'dwq_assessment_units': LayerAttributes(ATTRIBUTE, 'assessmentVal', 'assessmentSev', 'assessmentVal', 'assessmentSev', ['status2006']),
-        'soil': LayerAttributes(ATTRIBUTE, 'soilVal', 'soilSev', 'soilVal', 'soilSev', ['tex_def']),
+        'soil': LayerAttributes(ATTRIBUTE, 'soilVal', 'soilSev', 'soilVal', 'soilSev', ['musurftexgrp']),
         'shallow_ground_water': LayerAttributes(ATTRIBUTE, 'shallowWaterVal', 'shallowWaterSev', 'shallowWaterVal', 'shallowWaterSev', ['depth']),
-        'census_tracts_2010': LayerAttributes(ATTRIBUTE, 'censusVal', 'censusSev', 'censusVal', 'censusSev', ['pop100', 'arealand']),
+        'census_tracts_2020': LayerAttributes(ATTRIBUTE, 'censusVal', 'censusSev', 'censusVal', 'censusSev', ['pop100', 'arealand']),
```
Comment (steveoh) [inline src/tank_risk_tool.pyt]: **issue**: these fields do not exist on the tracts at the moment.

**Review by steveoh — COMMENTED** [review]
Does this look like the way things should be changing so far?

**Comment by nathankota** [comment]
> Does this look like the way things should be changing so far?

It does. I'm going to share the files changed page of this PR with Rick S. to see if the changes make sense to him as well.

File: .vscode/settings.json [hunk 2]
```diff
@@ -9,14 +9,22 @@
     "${workspaceRoot}/setup.cfg"
   ],
   "cSpell.words": [
-    "FACILITYUST",
+    "agrc",
     "aprx",
     "arcpy",
+    "DATASETNAME",
     "facilityid",
+    "FACILITYUST",
```
Comment (nathankota) [inline .vscode/settings.json:17]: should this be FacilityPST since we're replacing FacilityUST?

Comment (steveoh) [reply .vscode/settings.json:17]: this is only a spelling dictionary so it doesn't really matter

File: src/tank_risk_tool.pyt (hunk 1 above)
Comment (steveoh) [reply src/tank_risk_tool.pyt]: this was added. thanks greg.

**Pull request merged**